history.md

```diff
@@ -303,7 +302,7 @@ This is definitely a huge step forward, but this game is nevertheless not good.
 | --- | --- | --- |
 | --- | JA2 Stracciatella team | unofficial patch and modding platform for JA2 |
 
-The team behind JA2 Stracciatella has regrouped and the project has moved to GitHub. A lot is happening right now and lots of progress is being made. Let's see where the future will get *JA2* and the Stracciatella project!
+The team behind JA2 Stracciatella has regrouped and the project has moved to GitHub. A lot is happening right now and lots of progress is being made. The new team managed to release several new versions, the latest being 0.16.0: Apart from several bugfixes, this version also has some great new features, including a new game mode, exclusive to *JA2 Stracciatella*. Let's see where the future will get *JA2* and the Stracciatella project!
```

Comment:
It's better to keep this generic, otherwise we'd have to update it with each release. But you can insert a link to the features instead: https://ja2-stracciatella.github.io/#features